### ISSUE TITLE
Opening a menu while wallwalking releases not toggled wallwalk.

### ISSUE THREAD

**Issue by illwieckz** [Unvanquished/Unvanquished#569]
Hi, this is a very old issue from Tremulous 1.1 and always true in Unvanquished 0.36.0, if wallwalk is not toggled (so you must press a key to maintain wallwalk), when you open the build menu you fall, even if you are maintaining the wallwalk key.

**Comment by overvolting** [comment]
who doesn't toggle wallwalk? 8-$

**Comment by illwieckz** [comment]
I never toggled wallwalk. :stuck_out_tongue: I use the shift key under my auricular to wallwalk and the space bar under my inch to jump so I can wallwalk then jump while having index, middle and ring fingers free on WASD to move and change direction. I disabled wallwalk toggling in 2006 the first time I played to Tremulous, then I reactivated wallwalk toggling for the first time right now to write this issue and to verify that this issue affects only not-toggled wallwalk. I can't imagine playing with toggled wallwal, it's too late for me to learn. :sweat_smile:
